packages/jest-changed-files/src/git.js

```diff
@@ -46,33 +46,44 @@ const findChangedFilesUsingCommand = async (
 const adapter: SCMAdapter = {
   findChangedFiles: async (
     cwd: string,
+    roots: Array<Path>,
     options?: Options,
```

Comment:
How about moving `roots` to `Options`? This way we could keep it backwards compatible.

Reply:
is the idea that we would keep `roots` as optional within `Options` but ensure that `getChangedFilesForRoots` always provides it?

Reply:
Yup, ideally with the default behavior being the same as current one (pre this PR)

Reply:
Just pushed the fix. Let me know what you think. Thanks!